languages/rust/concepts/structs/about.md

```diff
@@ -1 +1,23 @@
-TODO: add information on structs concept
+This concept exercise gave you some practice writing structs and methods in Rust.
```

Comment:
Rewrite to avoid all "this exercise" references.

Reply:
Could you perhaps suggest a proper wording?

Reply:
Not at this moment, because I was just thinking of rewriting it entirely, which is non-trivial. 

Some of this text could go toward a "mutability" concept. The ideas that we really need to retain here are:

- structs collect a heterogenous set of data into a single named unit
- each member of a struct is called a **field**
- each member of a struct has a particular name by which it can be referred
- visibility is tracked at the field level
- there exist other, more specialized struct types.

Reply:
Not sure if it's more appropriate to talk more about the other struct types here or in a separate concept. Briefly

- Tuple structs are like structs, but the fields are anonymous; fields are accessed via packing/unpacking or a numeric index
- Unit structs are **zero-size types** (another concept?) which typically exist for the purpose of implementing traits.

Reply:
Another bit of discussion for which I'm not sure of the appropriate location: 

- structs are named units with named fields
- tuple structs are named units with anonymous fields
- tuples are anonymous units with anonymous fields
- anonymous structs, anonymous units with named fields, exist in certain languages (i.e. go) but not in Rust.

Reply:
Wow, those are excellent points. I'll leave you and the other Rust maintainers to it (or did you want my feedback?).

Reply:
I'm happy to take any feedback you have, but in general I feel it's the Rust team's responsibility to get this all polished.

Reply:
I agree. Let me know when you need any help.